proxysql/README.md

```diff
@@ -0,0 +1,111 @@
+# Agent Check: ProxySQL
+
+## Overview
+
+This check monitors [ProxySQL][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+
+### Installation
+
+The ProxySQL integration is included in the [Datadog Agent][3] package, so you don't need to install anything else on your servers.
+
+### Configuration
+
+#### Host
+1. Edit the `proxysql.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][4] to start collecting your ProxySQL performance data. See the [sample proxysql.d/conf.yaml][5] for all available configuration options.
+
+2. [Restart the Agent][6].
+
+##### Log Collection
+
+1. Collecting logs is disabled by default in the Datadog Agent, enable it in your `datadog.yaml` file:
+
+   ```yaml
+   logs_enabled: true
+   ```
+
+2. Add the log files you are interested in to your `proxysql.d/conf.yaml` file to start collecting your ProxySQL logs:
+
+   ```yaml
+     logs:
+         # Default logging file
+       - type: file
+         path: /var/log/proxysql.log
+         source: proxysql
+         service: <SERVICE_NAME>
+         # Logged queries, file needs to be in JSON
+         # https://github.com/sysown/proxysql/wiki/Query-Logging
+       - type: file
+         path: <QUERY_LOGGING_FILE_PATH>
+         source: proxysql
+         service: <SERVICE_NAME>
+         # Audit log
+         # https://github.com/sysown/proxysql/wiki/Audit-log
+       - type: file
+         path: <AUDIT_LOG_FILE_PATH>
+         source: proxysql
+         service: <SERVICE_NAME>
```

Comment:
Using double quotes here to avoid breaking the formatting

```suggestion
   ```yaml
     logs:
         # Default logging file
       - type: file
         path: /var/log/proxysql.log
         source: proxysql
         service: "<SERVICE_NAME>"
         # Logged queries, file needs to be in JSON
         # https://github.com/sysown/proxysql/wiki/Query-Logging
       - type: file
         path: "<QUERY_LOGGING_FILE_PATH>"
         source: proxysql
         service: "<SERVICE_NAME>"
         # Audit log
         # https://github.com/sysown/proxysql/wiki/Audit-log
       - type: file
         path: "<AUDIT_LOG_FILE_PATH>"
         source: proxysql
         service: "<SERVICE_NAME>"
```